static/app/__mocks__/react-date-range.tsx

```diff
@@ -0,0 +1,97 @@
+import type {CalendarProps, DateRangeProps, Range, RangeKeyDict} from 'react-date-range';
```

Comment:
Only comment I have is that I might leave a comment at the top of this file explaining __why__